mkdocs.yml

```diff
@@ -109,7 +109,7 @@ nav:
               - data_management/lts/tutorial/index.md
               - LTS and s3cmd Workflow: data_management/lts/tutorial/individual_lts_tutorial.md
           - Interfacing with LTS: data_management/lts/interfaces.md
-          - Bucket Permissions: data_management/lts/policies.md
+          - Access Control: data_management/lts/policies.md
```

Comment:
Change to "Identity and Access Management".

Rename the page to be `data_management/lts/iam_and_policies.md`

Be sure to add a redirect from the old page to the new page in the redirects section. When you add the redirect, be sure to put it in alphabetical order with the rest. I'd like to keep them sorted.